### PR TITLE
Environment Variable Support for UpdateKey Program

### PR DESCRIPTION
**Pull request by ljk662**
This PR adds the ability to use environment variables instead of user input for the secret arguments in the Update Key program.

### CODE REVIEW

File: feeder/src/updateKey.ts [hunk 1]
```diff
@@ -2,23 +2,30 @@ import * as keystore from './keystore'
...
 
 export async function updateKey(filePath: string): Promise<void> {
-  const password = await promptly.password(`Enter a passphrase to encrypt your key to disk:`, {
-    replace: `*`,
-  })
-  const confirm = await promptly.password(`Repeat the passphrase:`, { replace: `*` })
+  let password = process.env['PASSPHRASE'] || ''
+  let mnemonic = process.env['MNEMONIC'] || ''
+
+  if (password == '') {
```
Comment (hanjukim) [inline feeder/src/updateKey.ts:8]: could you change == to === ?

File: feeder/src/updateKey.ts [hunk 2]
```diff
@@ -2,23 +2,30 @@ import * as keystore from './keystore'
...
 
   if (password.length < 8) {
     console.error(`ERROR: password must be at least 8 characters`)
     return
   }
 
-  if (password !== confirm) {
-    console.error(`ERROR: passphrases don't matchPassword confirm failed`)
-    return
+  if (mnemonic == '') {
```
Comment (hanjukim) [inline feeder/src/updateKey.ts:25]: This too.

**Review by hanjukim — CHANGES_REQUESTED** [review]

File: feeder/src/updateKey.ts (hunk 1 above)
Comment (ljk662) [reply feeder/src/updateKey.ts]: done

File: feeder/src/updateKey.ts (hunk 2 above)
Comment (ljk662) [reply feeder/src/updateKey.ts]: done

**Review by hanjukim — APPROVED** [review]

**Comment by hanjukim** [comment]
Thank you!

**Pull request merged**